packages/nimble/inst/CppCode/predefinedNimbleLists.cpp

```diff
@@ -246,12 +247,14 @@ void OptimControlNimbleList::copyFromSEXP(SEXP S_nimList_) {
     SET_STRING_ELT(S_pxData, 0, mkChar(".xData"));
     PROTECT(S_trace = findVarInFrame(GET_SLOT(S_nimList_, S_pxData),
```

Comment:
I've merged the optim work and am addressing the independent `PROTECT` issue in #406